### PR TITLE
Handle count zero for group member api call

### LINKED ISSUE #1204: SCIM group member resource returns a 500 error code when passing count=0 as a parameter
Using the API, to work with pagination it is sometimes useful to read just the number of resources. The quickest and more efficient way is to issue a get request specifying `0` as the `count` url parameter, like:

```console
$ curl -s -H "Autho..." https://iam-base-url/scim/Groups?count=0
{"totalResults":55,"schemas":["urn:ietf:params:scim:api:messages:2.0:ListResponse"],"Resources":[]}
```

While this works fine with the `scim/Groups` endpoint, it does not with the `scim/Groups/{id}/members` one. In the latter case, the server responses with a `500 Server Error`

To replicate:

```
curl -s -H "Autho..." https://iam-base-url/scim/Groups/{id}/members?count=0
```

Tested on 1.13.4.

### PR DESCRIPTION
**Pull request by garaimanoj**
No more 500 error with count=0

image

### CODE REVIEW

**Review by rmiccoli — APPROVED** [review]

File: iam-login-service/src/test/java/it/infn/mw/iam/test/scim/group/ScimGroupManagerTests.java [hunk 1]
```diff
@@ -129,6 +129,16 @@ void roleReaderCanSeeGroupMembers() throws Exception {
...
 
+  @Test
+  @WithMockUser(username = "test", roles = "READER")
+  void roleReaderCanSeeGroupMembersCount() throws Exception {
+    mvc.perform(get(GROUP_URI + "/{uuid}/members?count=0", TEST_001_GROUP_ID).content(SCIM_CONTENT_TYPE))
+      .andExpect(status().isOk())
+      .andExpect(content().contentType(SCIM_CONTENT_TYPE))
+      .andExpect(jsonPath("$.totalResults", equalTo(0)))
+      .andExpect(jsonPath("$.Resources").isArray());
+  }
```
Comment (enricovianello) [inline iam-login-service/src/test/java/it/infn/mw/iam/test/scim/group/ScimGroupManagerTests.java:141]: We could add another call specifying Analisys or Production group uuid because these groups have a non-empty amount of members

**Review by enricovianello — CHANGES_REQUESTED** [review]
Just a copy and paste in the test, in order to add a test for a non empty group.

File: iam-login-service/src/test/java/it/infn/mw/iam/test/scim/group/ScimGroupManagerTests.java (hunk 1 above)
Comment (garaimanoj) [reply iam-login-service/src/test/java/it/infn/mw/iam/test/scim/group/ScimGroupManagerTests.java:141]: Done

**Review by enricovianello — APPROVED** [review]
